contrib/python/pypodman/pypodman/lib/actions/images_action.py

```diff
@@ -37,7 +37,7 @@ def __init__(self, args):
 
         self.columns = OrderedDict({
             'name':
-            ReportColumn('name', 'REPOSITORY', 40),
+            ReportColumn('name', 'REPOSITORY', 0),
```

Comment:
Is it a python3-ism that you don't need to supply all the values (missing one for `default` here)?

Reply:
ReportColumn() defaults missing arguments to None

Reply:
luv me my python for this, although sometimes I hate it for this too!